modules/search/instant-search/components/search-app.jsx

```diff
@@ -21,38 +21,55 @@ import {
 	getFilterQuery,
 	getSearchQuery,
 	getSortQuery,
+	getResultFormatQuery,
 	hasFilter,
 	restorePreviousHref,
 } from '../lib/query-string';
 import { removeChildren, hideElements, hideChildren, showChildren } from '../lib/dom';
+import { RESULT_FORMAT_MINIMAL } from '../lib/constants';
 
 class SearchApp extends Component {
 	static defaultProps = {
-		resultFormat: 'minimal',
 		widgets: [],
 	};
 
 	constructor() {
 		super( ...arguments );
 		this.input = createRef();
-		this.state = { isLoading: false, response: {}, requestId: 0, showResults: false };
+		this.state = {
+			isLoading: false,
+			response: {},
+			requestId: 0,
+			showResults: false,
+			resultFormat: RESULT_FORMAT_MINIMAL,
```

Comment:
I've used state to start with a default value and then change it if the user has specified something different in the query string, but I could probably put that logic into `getResultFormatQuery` instead. Thoughts @jsnmoon?

Reply:
Ya, this should eventually get hooked up somewhere in the admin. Maybe not the widget since the user should be able to configure search without a widget.